tempodb/compaction_block_selector.go

```diff
@@ -54,6 +56,11 @@ func newTimeWindowBlockSelector(blocklist []*backend.BlockMeta, maxCompactionRan
 		MaxBlockBytes:        maxBlockBytes,
 	}
 
+	if maxCompactionRange == 0 {
```

Comment:
i believe this returns an invalid block selector which would make the compactor behave unpredictably.

perhaps we should make a `validateConfig` function like this:

https://github.com/grafana/tempo/blob/main/tempodb/config.go#L119

and then call it in `EnableCompaction`:

https://github.com/grafana/tempo/blob/main/tempodb/tempodb.go#L379

Reply:
So I need to check that compactor.compaction.compaction_window is not zero. If it is, I log it as an error. But as I would just log, and not temrinate the program, that divison code would still execute, isn't?